os/fs/driver/mtd/smart.c

```diff
@@ -1041,7 +1041,7 @@ static int smart_setsectorsize(FAR struct smart_struct_s *dev, uint16_t size)
 	
 #ifdef CONFIG_MTD_SMART_JOURNALING
 	/** Journal Sector is reserved at the last of smartfs partition, it doesn't use MTD Header.
-	  * Journal sectors divided to 2 area, this will be toggled if anyone is filled with data,
+	  * Journal sectors divided to 2 area, this will be toggled if one area is filled with data,
```

Comment:
we don't use area now, could you please remove this description? I forgot it.

Reply:
Ohkay